cinema/serializers.py

```diff
@@ -18,7 +18,56 @@ def update(self, instance, validated_data):
             "description", instance.description
         )
         instance.duration = validated_data.get("duration", instance.duration)
+        instance.save()
+        return instance
+
+
+class ActorSerializer(serializers.Serializer):
```

Comment:
I think it would be better to use `ModelSerializer`s here. But I guess you already did it in the next task, so it's probably not necessary to fix it here as well. _I guess._

Reply:
in this task, it's not a mistake. but I prefer using ModelSerializer